### ISSUE TITLE
time zone

### ISSUE THREAD

**Issue by akapucu** [chrissanders/FlowBAT#7]
Is it possible to disable UTC time zone and get everything on eastern time 
Searching is very difficult when i need to convert time

**Comment by DenisGorbachev** [comment]
@akapucu Yes, it's possible: just click the "UTC" button on the query page. Here's a screenshot:

![UTC](https://cloud.githubusercontent.com/assets/829578/5938431/a34b2284-a72d-11e4-93a2-38164c3be780.png)

Let me know if you have any other questions.

**Comment by chrissanders** [comment]
I believe @akapucu is asking if it is possible to search based on local time instead of UTC. Currently, that's not supported. Although, that is something we will consider adding in as an option.

**Comment by akapucu** [comment]
Thats correct Chris,

Thank you Denis

________________________________
